codecov.hcl

```diff
@@ -0,0 +1,30 @@
+description = "Codecov Report Uploader"
+binaries = ["codecov"]
+test = "codecov --version"
+version "v0.3.2" {}
+
+linux {
+  source = "https://github.com/codecov/uploader/releases/download/${version}/codecov-alpine"
+
+  on "unpack" {
+    rename {
+      from = "${root}/codecov-alpine"
+      to = "${root}/codecov"
+    }
+  }
+}
+
+darwin {
+  source = "https://github.com/codecov/uploader/releases/download/${version}/codecov-macos"
+
+  on "unpack" {
+    rename {
+      from = "${root}/codecov-macos"
+      to = "${root}/codecov"
+    }
+  }
+}
+
+sha256sums = {
+  "https://github.com/codecov/uploader/releases/download/${version}/codecov-alpine": "698001de6a275407f4f9a71ea64158c92e54858b390579560fb4007e6b28116c"
```

Comment:
`${version}` shouldn't be in the sha256sums URLs - it should be the fully interpolated URL to the artefact.

Reply:
codecov-macos seems to be also missing. You can automatically fill this in with:

```
hermit manifest add-digests codecov.hcl
```